sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusMetricsReceiver.cs

```diff
@@ -0,0 +1,195 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.ServiceBus.Listeners;
+using Microsoft.Azure.WebJobs.ServiceBus;
+using System;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Azure.Messaging.ServiceBus;
+using Azure.Messaging.ServiceBus.Administration;
+using Microsoft.Azure.WebJobs.Extensions.ServiceBus.Config;
+
+namespace Microsoft.Azure.WebJobs.Extensions.ServiceBus.Listeners
+{
+    internal class ServiceBusMetricsReceiver
```

Comment:
This class will be called directly by the ScaleControllerRedesign, please make `public`

Reply:
Because it was internal, I wasn't too concerned with its public interface. For any surface area we decide we need to promote to public, we'll need to do a much more thorough review/design to ensure we're only exposing what is needed, and that the APIs are clean and future proof.

Reply:
I changed the `ServiceBusMetricsReceiver` ctor so it can stay internal